tests/explorer.api.tests/ExploreTests.cs

```diff
@@ -59,24 +64,52 @@ public void SuccessWithContents()
             });
         }
 
-        [Theory]
-        [InlineData("")]
-        [InlineData("/")]
-        [InlineData("/invalid endpoint test")]
-        public void FailWithBadEndPoint(string endpoint)
+        [Fact]
```

Comment:
Why change this? `Theory` doesn't work with the vcr I guess?

Reply:
Yes, the problem here is that there were multiple tests using the same VCR cassette file and this didn't work out well.